bundles/org.openhab.binding.solarman/README.md

```diff
@@ -0,0 +1,300 @@
+# Solarman Logger Binding
+
+[![Build Status](https://app.travis-ci.com/catalinsanda/org.openhab.binding.solarman.svg?branch=main)](https://app.travis-ci.com/catalinsanda/org.openhab.binding.solarman)
+
+Binding used to communicate with Solarman (IGEN-Tech) v5 based solar inverter data loggers in direct-mode over the local
+network.
+More information about the different types of stick loggers is available on the
+[Solarman](https://www.solarmanpv.com/products/data-logger/stick-logger/) site.
+
+These data loggers are used by inverters from a lot of manufacturers, just to name a few:
+
+- Deye
+- Sofar
+- Solis
+- ZCS Azzurro
+- KStar
+
+## Supported Things
+
+The `solarman:logger` thing supports reading data from a Solarman LSW-3 Stick Logger (it might also work with LSE-3 and
+maybe others)
+when connected to a supported inverter.
```

Comment:
```suggestion
The `solarman:logger` thing supports reading data from a Solarman LSW-3 Stick Logger (it might also work with LSE-3 and maybe others) when connected to a supported inverter.
```